mne/preprocessing/stim.py

```diff
@@ -67,3 +79,108 @@ def eliminate_stim_artifact(raw, events, event_id, tmin=-0.005,
         elif mode == 'window':
             raw[picks, first_samp:last_samp] = data * window[np.newaxis, :]
     return raw
+
+
+def fix_stim_artifact(inst, events=None, event_id=None, tmin=-0.2,
+                      tmax=-0.1, mode='linear'):
+    """Eliminates stimulations artifacts from instance
+
+    The instance will be modified in place (no copy)
```

Comment:
if you return the instance you should have a copy param that is false by default ie. copy by default. You can just do

```
if copy:
    inst = inst.copy()
```